notebooks/nessie-iceberg-flink-demo-nba.ipynb

```diff
@@ -0,0 +1,719 @@
+    "\n",
+    "# Creating `salaries` table\n",
+    "(table_env.connect(FileSystem().path('../datasets/nba/salaries.csv'))\n",
+    "  .with_format(OldCsv()\n",
+    "               .field('Season', DataTypes.STRING()).field(\"Team\", DataTypes.STRING())\n",
+    "               .field(\"Salary\", DataTypes.STRING()).field(\"Player\", DataTypes.STRING()))\n",
+    "  .with_schema(Schema()\n",
+    "               .field('Season', DataTypes.STRING()).field(\"Team\", DataTypes.STRING())\n",
+    "               .field(\"Salary\", DataTypes.STRING()).field(\"Player\", DataTypes.STRING()))\n",
+    "  .create_temporary_table('dev_catalog.nba.salaries_temp'))\n",
+    "\n",
+    "table_env.execute_sql(\"\"\"CREATE TABLE IF NOT EXISTS dev_catalog.nba.salaries\n",
+    "            (Season STRING, Team STRING, Salary STRING, Player STRING)\"\"\").wait()\n",
+    "\n",
+    "tab = table_env.from_path('dev_catalog.nba.salaries_temp')\n",
+    "tab.execute_insert('dev_catalog.nba.salaries').wait()\n",
+    "\n",
+    "# Creating `totals_stats` table\n",
+    "(table_env.connect(FileSystem().path('../datasets/nba/totals_stats.csv'))\n",
+    "  .with_format(OldCsv()\n",
+    "               .field('Season', DataTypes.STRING()).field(\"Age\", DataTypes.STRING()).field(\"Team\", DataTypes.STRING())\n",
+    "               .field(\"ORB\", DataTypes.STRING()).field(\"DRB\", DataTypes.STRING()).field(\"TRB\", DataTypes.STRING())\n",
+    "               .field(\"AST\", DataTypes.STRING()).field(\"STL\", DataTypes.STRING()).field(\"BLK\", DataTypes.STRING())\n",
+    "               .field(\"TOV\", DataTypes.STRING()).field(\"PTS\", DataTypes.STRING()).field(\"Player\", DataTypes.STRING())\n",
+    "               .field(\"RSorPO\", DataTypes.STRING()))\n",
+    "  .with_schema(Schema()\n",
+    "               .field('Season', DataTypes.STRING()).field(\"Age\", DataTypes.STRING()).field(\"Team\", DataTypes.STRING())\n",
+    "               .field(\"ORB\", DataTypes.STRING()).field(\"DRB\", DataTypes.STRING()).field(\"TRB\", DataTypes.STRING())\n",
+    "               .field(\"AST\", DataTypes.STRING()).field(\"STL\", DataTypes.STRING()).field(\"BLK\", DataTypes.STRING())\n",
+    "               .field(\"TOV\", DataTypes.STRING()).field(\"PTS\", DataTypes.STRING()).field(\"Player\", DataTypes.STRING())\n",
+    "               .field(\"RSorPO\", DataTypes.STRING()))\n",
+    "  .create_temporary_table('dev_catalog.nba.totals_stats_temp'))\n",
+    "\n",
+    "table_env.execute_sql(\n",
+    "        \"\"\"CREATE TABLE IF NOT EXISTS dev_catalog.nba.totals_stats (Season STRING, Age STRING, Team STRING,\n",
+    "        ORB STRING, DRB STRING, TRB STRING, AST STRING, STL STRING, BLK STRING, TOV STRING, PTS STRING,\n",
+    "        Player STRING, RSorPO STRING)\"\"\").wait()\n",
+    "\n",
+    "tab = table_env.from_path('dev_catalog.nba.totals_stats_temp')\n",
+    "tab.execute_insert('dev_catalog.nba.totals_stats').wait()\n"
```

Comment:
Let's print something (number of rows, or `Success!`) here. Otherwise it just emits WARNINGs in red in mybinder.

Reply:
Current output:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.hadoop.security.authentication.util.KerberosUtil (file:/srv/conda/envs/flink-demo/lib/python3.7/site-packages/pyflink/lib/hadoop-auth-2.10.1.jar) to method sun.security.krb5.Config.getInstance()
WARNING: Please consider reporting this to the maintainers of org.apache.hadoop.security.authentication.util.KerberosUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

Reply:
added some output